wiki/Music_theory/Rhythm/de.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+---
+
+# Rhythmus
+
+**Rhythmus** ist "die Anordnung von Tönen, Stille und Betonung in einem Song".[^rhythm-ref] Beim Erstellen von [Beatmaps](/wiki/Beatmap) achten Mapper darauf, wo Töne in der Musik erscheinen, damit sie wissen, wo [Hit-Objekte](/wiki/Gameplay/Hit_object) und [Hitsounds](/wiki/Beatmapping/Hitsound) platziert werden sollen, um dem Rhythmus zu folgen und bestimmte Teile davon zu betonen. Als Rhythmusspiel ist Rhythmus ein fester Bestandteil jeder Beatmap in osu!.
```

Comment:
`"die Anordnung von Tönen, Stille und Betonung in einem Song"`
Should probably mention it's translated from the original English version (maybe in the footnote).

Reply:
Not really needed in my opinion as the title already lets the reader assume the article is in English. If you still prefer to have it, I would write something like this (based on the [rule for citations](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing#blockzitate))

```md
[^rhythm-ref]: ["Understanding Rhythm in Music: 7 Elements of Rhythm" auf MasterClass](https://www.masterclass.com/articles/understanding-rhythm-in-music) (aus dem Englischen übersetzt)
```

Reply:
I would prefer it, yes. It seems like good sense to include a note like this somewhere, as the translation is an interpretation on your part, which means it's not technically a direct quote anymore.